public/js/admin.js

```diff
@@ -174,8 +171,8 @@ var mcAnswerCount = 4;
 var addMCAnswers = function(dom){
     mcAnswerCount++;
      var newdiv = document.createElement('div');
```

Comment:
Indentation

Reply:
@ahmedqarmout did you just tell yourself to fix your own PR? xD

Reply:
fixed it